.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```

Comment:
- [ ] As required for the project, Kindly make sure you correctly update your linters config as right now you should have 5 checks for linters. And to to do that you should update the content of your `.github/workflows/linters.yml` file by using the content from the **[template for linters configurarion](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml)**. For details you need about linters config for this project, you can refer this **[HTML-CSS-JS](https://github.com/microverseinc/linters-config/tree/master/html-css-js)** repo.

Reply:
okay, Thank you